tests/Storage/Compression/Algorithms/BZIP2Test.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Utopia\Tests;
```

Comment:
Would you please use:

```suggestion
namespace Tests\Storage\Compression\Algorithms;
```

to match psr-4 standards?